core/frontend/src/libs/settings.ts

```diff
@@ -33,6 +33,19 @@ class Settings {
   updateVersionUpdateNotificationTime() {
     settingsStore.updateVersionUpdateNotificationTime()
   }
+
+  // eslint-disable-next-line
+  run_tour_version(version: number) : Promise<void> {
+    return new Promise((resolve) => {
+      const store_tour_version = settingsStore.tour_version
+      if (version === store_tour_version) {
+        throw new Error(`Tour version (${version}) is the same as in store (${store_tour_version}})`)
```

Comment:
do we really need to throw an exception here? this is throwing when opening the page after first time.